dev-docs/howtoguides/run_dep8_tests.md

```diff
@@ -0,0 +1,24 @@
+# How to run dep8 tests
+
+Our package currently supports [dep8
+tests](https://salsa.debian.org/ci-team/autopkgtest/raw/master/doc/README.package-tests.rst).
```

Comment:
Can we link to the rendered version instead? https://salsa.debian.org/ci-team/autopkgtest/-/blob/master/doc/README.package-tests.rst